targets/opsis_base.py

```diff
@@ -251,9 +251,16 @@ def __init__(self, platform,
             self.register_sdram_phy(self.ddrphy)
 
         self.specials += Keep(self.crg.cd_sys.clk)
-        platform.add_platform_command("""
-NET "{sys_clk}" TNM_NET = "GRPsys_clk";
-""", sys_clk=self.crg.cd_sys.clk)
+        self.specials += Keep(self.crg.cd_base50.clk)
+        platform.add_platform_command(
+            """
+NET "{sys_clk}" TNM_NET = "GRPsys_clk"; # Only used for PERIOD constraint
```

Comment:
Are you sure we really need this? Also we will have to use something like this for false paths:
https://github.com/m-labs/migen/blob/master/migen/build/xilinx/ise.py#L201